railgun.yml

```diff
@@ -0,0 +1,18 @@
+# https://development.shopify.io/tools/dev/railgun/Railgun-Config
```

Comment:
Since it's a public repo I'd just remove this. Not a big deal either way but still.